packages/tools/src/eventDispatchers/shared/getActiveToolForMouseEvent.ts

```diff
@@ -39,6 +43,8 @@ export default function getActiveToolForMouseEvent(
 
   const toolGroupToolNames = Object.keys(toolGroup.toolOptions);
   const defaultMousePrimary = toolGroup.getDefaultMousePrimary();
+  const mouseButton =
```

Comment:
I'm not 100% keen on adding an override here, but overall I think it is the cleanest solution since it allows other event types to pretend to be mouse events, which is something I think will be useful generally.